.travis.yml

```diff
@@ -176,21 +181,6 @@ jobs:
     - <<: *FULL_BUILD_JOB
       env: MODULE='cql' ARGS=' -Pcassandra3-murmur-ssl -Dcassandra.docker.version=3.0.18 -Dtest=**/diskstorage/cql/CQLStoreTest.java'
 
-    - stage: test
```

Comment:
Could you please explain why you removed this and what is this?

Reply:
It was executed one or two year ago the last time. It refers to https://scan.coverity.com/projects/janusgraph-janusgraph.

Reply:
I will revert these changes and create an issue for it. There are other depending changes.